plugins/inputs/net/net.go

```diff
@@ -64,6 +77,7 @@ func (s *NetIOStats) Gather(acc telegraf.Accumulator) error {
 	}
 
 	for _, io := range netio {
+		var iface net.Interface
```

Comment:
That's correct, and I didn't move it alongside the definition of iface, because in my testing I found we're not sure that the iface currently exists until this branch of the conditional (line 92)